File: app/src/main/java/com/hnidesu/taskmanager/utility/LogUtil.java
package com.hnidesu.taskmanager.utility;

import android.util.Log;

import androidx.annotation.StringRes;

import com.hnidesu.taskmanager.MyApplication;

public class LogUtil {
    public static final String TAG="TaskManager";
    public static void Log(String msg){
        Log.d(TAG,msg);
    }

    public static void Log(@StringRes int resId){
        Log.d(TAG,MyApplication.getInstance().getString(resId));
    }

    public static void Error(String msg,Throwable e){
        Log.e(TAG,msg,e);
    }

    public static void Error(@StringRes int resId, Throwable e){
        Log.e(TAG,MyApplication.getInstance().getString(resId),e);
    }
}

File: app/src/main/java/com/hnidesu/taskmanager/util/LogUtil.kt
package com.hnidesu.taskmanager.util

import android.util.Log

object LogUtil{
    private const val TAG = "TaskManager"
    fun debug(msg: String) {
        Log.d(TAG, msg)
    }

    fun error(msg: String?, e: Throwable?) {
        Log.e(TAG, msg, e)
    }
}
